.circleci/deployment/workflows.yml

```diff
@@ -37,21 +37,21 @@
           filters:
             branches:
               only:
-                - develop
+                - 1544-Nginx-with-domain # switch back to develop before merge
```

Comment:
TODO: To be removed when ready to merge, (only to hijack staging)

Reply:
reminder to get to this TODO